USA/FL/Bay/Court/scraper/tests/test_ScraperUtils.py

```diff
@@ -139,3 +152,24 @@ def test_parse_out_path_correct_length(self):
         filename_too_long = '01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789'
         parsed_path = ScraperUtils.parse_out_path(os.getcwd(), filename_too_long, 'txt')
         assert len(parsed_path) <= 256
+
+    @flagsaver.flagsaver(collect_pii=False)
+    def test_enforce_use_of_pii_wrappers(self):
+        with pytest.raises(TypeError, match=".*Pii.String.*"):
+            builder = ScraperUtils.RecordBuilder(
+                    id="foo", state="bar", county="baz")
+            builder.first_name = "I should be in a PII wrapper"
+            builder.build()
+
+        with pytest.raises(TypeError, match=".*Pii.String.*"):
+            ScraperUtils.Record(id="foo", state="bar", county="baz",
+                                first_name="I should be in a PII wrapper")
+
+    @flagsaver.flagsaver(collect_pii=False)
+    def test_happy_builder(self):
+        builder = ScraperUtils.RecordBuilder(
+                id="foo", state="bar", county="baz")
+        want = Pii.String("I'm in a PII wrapper")
+        builder.first_name = want
+        r = builder.build()
+        assert r.first_name == want
```

Comment:
Add unit tests for the expected behaviour if collect_pii=True?

Reply:
Done.